src/thunderbots/software/backend_output/grsim/motion_controller.cpp

```diff
@@ -0,0 +1,206 @@
+#include "backend_output/grsim/motion_controller.h"
+
+#include <chrono>
+#include <iostream>
+#include <utility>
+
+#include "ai/world/robot.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+#include "shared/constants.h"
+
+/**
+ *.cpp file for the grSim motion controller.
```

Comment:
nit: space between asterisks and the actual comments

Reply:
Added spaces.